ProteoformSuiteGUI/DisplayUtility.cs

```diff
@@ -57,19 +57,6 @@ public static void tooltip_graph_display(ToolTip t, MouseEventArgs e, Chart c, P
             { }           
         }
 
-        //public static void EditInputFileDGVs(DataGridView dgv, Purpose purpose)
```

Comment:
Yes I think I'll go through master and get rid of all TD stuff that's outdated after we meet